node/CMakeLists.txt

```diff
@@ -21,5 +33,4 @@ target_link_libraries(kagome
     )
 if (BACKWARD)
   add_backward(kagome)
-endif()
-kagome_install(kagome)
```

Comment:
Move to "else", if external-project-test requires target "kagome"

Reply:
It seems it is not required.